apps/stats-web/src/components/graph/Graph.tsx

```diff
@@ -25,74 +30,140 @@ const Graph: React.FunctionComponent<IGraphProps> = ({ rangedData, snapshotMetad
   const { resolvedTheme } = useTheme();
   const intl = useIntl();
   const graphTheme = getTheme(resolvedTheme);
-  const smallScreen = useMediaQuery(breakpoints.xs.mediaQuery);
-  const minValue = rangedData && snapshotMetadata.unitFn(rangedData.map(x => x.value).reduce((a, b) => (a < b ? a : b))).value;
-  const maxValue = snapshotData && snapshotMetadata.unitFn(rangedData.map(x => x.value).reduce((a, b) => (a > b ? a : b))).value;
+  
   const graphData = snapshotData
-    ? [
-        {
-          id: snapshot,
-          color: "rgb(1,0,0)",
-          data: rangedData
-            .map(_snapshot => ({
-              x: _snapshot.date,
-              y: roundDecimal(snapshotMetadata.unitFn(_snapshot.value).value)
-            }))
-            .sort(function (a, b) {
-              return Number(new Date(a.x)) - Number(new Date(b.x));
-            })
+    ? rangedData.map(_snapshot => (
+      {
+        time: moment.utc(_snapshot.date).format('YYYY-MM-DD'),
+        value: _snapshot.value
```

Comment:
There was a unitFn called in the previous code `y: roundDecimal(snapshotMetadata.unitFn(_snapshot.value).value)` which is probably why the values don't match.